modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini

```diff
@@ -444,6 +449,11 @@ SEK-STINA-COMPOUND = SEK-SIOR
 SEK-STINA-OIS-COMPOUND = SEK-SIOR
 SEK-STIBOR-OIS Compound = SEK-SIOR
 
+SEK-SWESTR = SEK-SWESTR
```

Comment:
```suggestion
# "SEK-SWESTR"/"SEK-SWESTR-OIS Compound" are the ISDA 2021 codes
SEK-SWESTR = SEK-SWESTR
```

Reply:
Good point - updated like DKK DESTR using -> https://www.isda.org/a/U0WgE/2006-FRO-Mapping-to-2021-FROs-Version-5.pdf